src/main/java/org/opentripplanner/routing/core/AStarRequest.java

```diff
@@ -34,6 +38,13 @@ public class AStarRequest {
   private final Envelope fromEnvelope;
   private final Envelope toEnvelope;
 
+  protected IntersectionTraversalCalculator intersectionTraversalCalculator = IntersectionTraversalCalculator.create(
+    IntersectionTraversalModel.SIMPLE,
+    DrivingDirection.RIGHT
+  );
```

Comment:
Should this be made the `IntersectionTraversalCalculator.DEFAULT` calculator. Easier to reuse, and avoids creating a new one for each request.

Reply:
Done